ledger/acctupdates_test.go

```diff
@@ -1766,22 +1767,22 @@ func TestCompactDeltasResources(t *testing.T) {
 	delta, _ := outResourcesDeltas.get(addrs[0], 100)
 	require.NotEmpty(t, delta.newResource)
 	require.True(t, !delta.newResource.IsApp() && !delta.newResource.IsAsset())
-	require.Equal(t, resourceFlagsNotHolding, delta.newResource.ResourceFlags)
+	require.True(t, !delta.newResource.IsHolding())
```

Comment:
Please revert to exact comparison here and below. These are important since IsHolding checks a flag but the prev statement checks exact value - i.e. no other flags are set.